experiments/modern/src/maki-interpreter/virtualMachine.js

```diff
@@ -287,6 +287,14 @@ function* interpret(start, program, stack = []) {
         twoArgOperator((b, a) => b >> a);
         break;
       }
+      // new
+      case 96: {
+        const classesOffset = command.arg;
+        const klass = classes[classesOffset];
+        const klassInst = new klass();
+        stack.push(klassInst);
+        break;
+      }
```

Comment:
Woot! New opcode incoming!

Reply:
might need tweaking, but seems to be as simple as just `new`ing the class...